### DISCUSSION TITLE
Page indicators - dots vs scroll bar

### DISCUSSION THREAD

**Discussion by SteveAmor** [InfiniTimeOrg/InfiniTime#2202] [Ideas]
Was there a particular reason to create a dot  page indicator for the Time and Date setting pages when all other apps/screens use the line page indicator?

**Comment by SteveAmor** [comment]
I see it was discussed here #1465 
To me, having dots is odd compared to the scroll bar being used everywhere else.

**Reply by mark9064** [reply]
Hmm yeah, feels inconsistent to me too
